src/data/common/Dhis2DataElement.ts

```diff
@@ -48,16 +48,20 @@ const dataElementFields = {
             name: true,
             categoryOptions: {
                 id: true,
+                code: true,
                 name: true,
                 shortName: true,
-                formName: true,
+                displayFormName: true,
             },
         },
         categoryOptionCombos: {
             id: true,
             name: true,
             shortName: true,
-            categoryOptions: true,
+            categoryOptions: {
+                id: true,
+                code: true,
+            },
```

Comment:
Do we need this? after the previous fix in the type, it shows no error, so I guess it's not used. (the form loads normally)